indra/newview/llpanelface.cpp

```diff
@@ -5098,12 +5102,49 @@ void LLPanelFace::LLSelectedTE::getPbrMaterialId(LLUUID& id, bool& identical, bo
             {
                 mHasFacesWithPBR = true;
             }
-            return pbr_id;
+            if (mInitialized)
+            {
+                if (mPBRId != pbr_id)
+                {
+                    mIdenticalId = false;
+                }
+                
+                LLGLTFMaterial* te_override = object->getTE(te_index)->getGLTFMaterialOverride();
+                if (te_override)
+                {
+                    LLGLTFMaterial override = *te_override;
+                    override.sanitizeAssetMaterial();
+                    mIdenticalOverride &= (override == mMaterialOverride);
+                }
+                else
+                {
+                    mIdenticalOverride &= (mMaterialOverride == LLGLTFMaterial::sDefault);
+                }
+            }
+            else
+            {
+                mInitialized = true;
+                mPBRId = pbr_id;
+                LLGLTFMaterial* override = object->getTE(te_index)->getGLTFMaterialOverride();
+                if (override)
```

Comment:
I think `LLGLTFMaterial::sDefault` is just the default constructor at this point, but as a precaution, we may consider assigning `mMaterialOverride` to  `LLGLTFMaterial::sDefault` here when `override` is null.

Reply:
Ok, will do, thank you.

Reply:
Actually it already is set to sDefault from contructor at string 5091 mMaterialOverride(LLGLTFMaterial::sDefault)